grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java

```diff
@@ -527,6 +671,14 @@ private void initMetrics() {
         GcpMetricsConstants.MILLISECOND,
         this,
         GcpManagedChannel::reportMaxUnresponsiveDrops);
+
+    createDerivedLongCumulativeTimeSeriesWithDirection(
+        GcpMetricsConstants.METRIC_CHANNEL_POOL_SCALING,
+        "The number of channels channel pool scaled up or down.",
```

Comment:
Why we need this metrics? It basically gives the same information as METRIC_NUM_CHANNELS?

Reply:
Do you mean the channel size can scale up and down but remain the same inside a scraping  period? What is the scraping  period?

Reply:
Yes. Reporting interval is usually 60 seconds but it is configurable.